src/System.Private.CoreLib/src/System/Globalization/FormatProvider.DateTimeFormat.cs

```diff
@@ -1,1213 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Diagnostics;
-using System.Text;
-using System.Threading;
-using System.Globalization;
-using System.Collections.Generic;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-using System.Security;
```

Comment:
Yes it is an unfinished experiment that I was doing earlier to try and provide an interface to the primitive types that we could substitute an invariant logic into the primitive layers that rely on IFormatProvider for parsing and tostring methods. I've not looked at this much recently so I leave it to you guys whether or not you want to maintain the abstraction.